tests/integration/05.Dfw.Tests.ps1

```diff
@@ -90,6 +90,9 @@ Describe "DFW" {
         $script:testvm1 = new-vm -name $testVMName1 @vmsplat
         $script:testvm2 = new-vm -name $testVMName2 @vmsplat
         $script:testvm3 = new-vm -name $testVMName3 @vmsplat
+        
+        Get-VM $testVMName1 | New-NetworkAdapter -StartConnected -Portgroup ($testls | Get-NsxBackingPortGroup).name | out-null
```

Comment:
alternative method could be to connect to any PG in the cluster the VM exists in and then use connect-nsxlogicalswitch - you dont need to care about backing PG then,...